packages/dappmanager/src/modules/release/ipfs/downloadRelease.ts

```diff
@@ -73,7 +74,7 @@ async function downloadReleaseIpfsFn(
       );
 
       // Pin release on visit
-      ipfs.pinAddNoThrow(hash);
+      if (db.ipfsClientTarget.get() === "local") ipfs.pinAddNoThrow(hash);
```

Comment:
Doing a db call in the middle of IPFS module breaks separation in concerns a lot. Is there any nicer way to achieve the same?